include/aspect/postprocess/visualization.h

```diff
@@ -124,6 +124,13 @@ namespace aspect
       class Interface
       {
         public:
+          /**
+           * Constructor. The constructor takes as argument the physical
+           * units of the quantity (scalar or vector-valued) computed by
+           * derived classes.
+           */
+          Interface (const std::string &physical_units);
```

Comment:
I just realized that we should probably add ="" as the default argument. Otherwise, we are not backwards compatible to existing user plugins.
@bangerth would it make sense to deprecate the default constructor to force users to think about units?

Reply:
I'm totally willing to not be backward compatible. Are we concerned that there are viz postprocessors out there that aren't in the main code base?

Reply:
That is what I meant by "user plugins" yes.

Reply:
Right. But what I'm asking whether you are worried that they exist, and that we need to be backward compatible?

Reply:
I don't feel strongly about it, but it is at least worth thinking about. One option would be to do:
```
DEAL_II_DEPRECATED Interface();
explicit Interface (const std::string &physical_units);
```
to make it a warning instead of an error.